client/src/components/Individuals/IndividualSpecies.jsx

```diff
@@ -0,0 +1,78 @@
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import IndividualsForm from "./IndividualsForm";
+import Loading from "../Loading";
+import fetchData from "../../utils";
+
+const IndividualSpecies = () => {
+  const { id } = useParams();
+
+  const [individualsOfSpecies, setIndividualsOfSpecies] = useState();
+
+  const getIndividualSpecies = async (speciesId) => {
+    try {
+      const response = await fetchData(`/individuals/${speciesId}`);
+
+      if (response) {
+        const {
+          data: { individuals },
+        } = response;
+
+        console.log(response, "response");
+        setIndividualsOfSpecies(individuals);
+      }
+    } catch (error) {
+      console.error("Error occured while fetching data", error);
+    }
+  };
+
+  console.log(individualsOfSpecies);
+
+  useEffect(() => {
+    getIndividualSpecies(id);
+  }, [id]);
+  return !individualsOfSpecies ? (
+    <Loading />
+  ) : individualsOfSpecies.length > 0 ? (
+    <div>
+      {individualsOfSpecies && (
+        <h2>{`List of Individual ${individualsOfSpecies[0].common_name}s`}</h2>
+      )}
+      <IndividualsForm setIndividualsOfSpecies={setIndividualsOfSpecies} />
+      {individualsOfSpecies.length > 0 && (
+        <div className="container">
+          {individualsOfSpecies.map((individual) => {
+            const {
```

Comment:
I think this could've been factored out into another component, just to look a little cleaner here.

Reply:
Oh good point! Done.